src/components/Connexion/Profile.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+import { matchPath } from 'react-router';
+import {
+  ListGroup,
+  ListGroupItem,
+  Row,
+  Container,
+  Col,
+  Nav,
+  NavLink,
+  NavItem,
+  Button,
+} from 'reactstrap';
+import Axios from 'axios';
+import './Profile.css';
+import avatar_placeholder from './placeholder-avatar.png'
+
+const Profile = (props) => {
+  const [profile, setProfile] = useState({});
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+
+   Axios({
+      method: 'GET',
+      url: `http://localhost:3000/api/users/${props.match.params.id}`,
```

Comment:
vous devez remplacer localhost:3000 par votre variable d'environnement sinon ça ne fonctionnera pas
```suggestion
      url: `http://process.env.REACT_APP_SERVER/api/users/${props.match.params.id}`,
```